pilot/pkg/features/ambient.go

```diff
@@ -46,9 +46,10 @@ var (
 		"If enabled, HBONE will be preferred when sending to destinations. ")
 
 	DefaultAllowFromWaypoint = registerAmbient(
-		"PILOT_DEFAULT_ENFORCE_FROM_WAYPOINT",
+		"PILOT_AUTO_ALLOW_WAYPOINT_POLICY",
```

Comment:
IIUC, the polarity of the flag changed but the default didn't?

Reply:
Correct.  We do need to make a decision to change to enable the flag before 1.22 goes out.

Reply:
@stevenctl Was this decision made?

Reply:
I don't think so. I personally would like to test a bit more as well.